astro/src/content/docs/apis/identity-providers/_facebook-post-request-body.mdx

```diff
@@ -84,6 +84,13 @@ import JSON from 'src/components/JSON.astro';
 
   <IdentityProviderTenantConfiguration idp_since={props.idp_since} />
 
+  <APIField name="identityProvider.tenantId" type="UUID" optional since="1.62.0">
+    The unique Id of the Tenant.
+
+    Providing a value creates an identity provider scoped to the specified tenant, otherwise a global identity provider is created. Tenant-scoped identity providers can only be used to authenticate in the context of the specified tenant while global identity providers can be used with any tenant.
```

Comment:
```suggestion
    The unique ID of the Tenant.

    Providing a value creates an identity provider scoped to the specified Tenant, otherwise a global identity provider is created. Tenant-scoped identity providers can only be used to authenticate in the context of the specified Tenant. Global identity providers can be used with any Tenant.
```
same as above